cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -74,25 +74,25 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
 
     private val projectNameOption by option(
         "--project-name",
-        help = "The speaking name of the project to download. For use together with '--project-url'. Will be ignored " +
-                "if '--ort-file' is also specified. (default: the last part of the project URL)"
+        help = "The speaking name of the project to download. For use together with '--project-url'. Is ignored if " +
```

Comment:
Nit: Personally, I'd either say "It is ignored if" or just "Ignored if", but not "Is ignored if".

Reply:
Done.